plume-models/src/blogs.rs

```diff
@@ -32,6 +32,7 @@ use {Connection, Error, Result, CONFIG};
 pub type CustomGroup = CustomObject<ApSignature, Group>;
 
 #[derive(Queryable, Identifiable, Clone, AsChangeset)]
+#[changeset_options(treat_none_as_null = "true")]
```

Comment:
I don't understand what this does, but apparently it did the trick

Reply:
It is an option for the `#[derive(AsChangeset)]`, that tells diesel to consider `Option` fields that are `None` to be considered as `NULL` in SQL, instead of not being included in the query at all.

Reply:
Ok, doesn't this potentially break other things that relied on it before in our code?

Reply:
not according to our extensive tests

Reply:
Test in production :tm:

Reply:
> Ok, doesn't this potentially break other things that relied on it before in our code?

It only affects the `AsChangeset` implementation, which is used when `UPDATE`ing blogs, so I think it is the only place we are doing that.